docs/5.api/rpc/providers.md

```diff
@@ -41,12 +40,43 @@ If you want to use a custom RPC provider with NEAR Wallet Selector, [check this
 
 | Provider                                                                   | Endpoint Root                                                |
 | -------------------------------------------------------------------------- | ------------------------------------------------------------ |
-| [NEAR](setup.md)                                                           | `https://rpc.testnet.near.org`                               |
-| [Pagoda](https://www.pagoda.co/console)                                    | `https://rpc.testnet.pagoda.co`                              |
+| [NEAR.org (deprecated)](setup.md)                                          | `https://rpc.testnet.near.org`                               |
 | [FASTNEAR](https://fastnear.com)                                           | `https://test.rpc.fastnear.com`                              |
 
 ## RPC Failover
 
 In `near-api-js` you can use [`FailoverRpcProvider`](../../4.tools/near-api-js/quick-reference.md#rpc-failover) to automatically switch RPC providers when one provider is experiencing downtime, or implement an RPC selection widget that allows users to add their own RPC provider.
 
 As a user, if a dApp or wallet doesn't support RPC failover and the primary provider is down, you can use an RPC Selector browser extension to redirect all requests to an RPC provider of your choice.
+
+## On NEAR.org RPC Deprecation
+
+Please read the following announcement: [Future of Pagoda Services](https://docs.near.org/blog/2024-08-13-pagoda-services).
+
+> The Infrastructure Committee feels that Pagoda's fully-subsidized near.org RPC service is getting in the way of decentralization efforts and is preventing high-quality commercial RPC offerings from gaining traction. If a NEAR core team continues to support a free-to-use near.org RPC service, it will be required to gradually lower its rate limits over the coming months to prevent abuse. More details on this plan will be communicated by the end of September 2024. In light of this proposed change, high-traffic near.org RPC users should start making plans to switch to other RPC providers.
```

Comment:
```suggestion
> The Infrastructure Committee feels that Pagoda's fully-subsidized near.org RPC service is getting in the way of decentralization efforts and is preventing high-quality commercial RPC offerings from gaining traction. If a NEAR core team continues to support a free-to-use near.org RPC service, it will be required to gradually lower its rate limits over the coming months to prevent abuse. In light of this proposed change, high-traffic near.org RPC users should start making plans to switch to other RPC providers.
```